javascript/ql/src/Quality/UnhandledStreamPipe.ql

```diff
@@ -0,0 +1,303 @@
+/**
+ * @id js/nodejs-stream-pipe-without-error-handling
+ * @name Node.js stream pipe without error handling
+ * @description Calling `pipe()` on a stream without error handling will drop errors coming from the input stream
+ * @kind problem
```

Comment:
```suggestion
 * @id js/unhandled-error-in-stream-pipeline
 * @name Unhandled error in stream pipeline
 * @description Calling `pipe()` on a stream without error handling will drop errors coming from the input stream
 * @kind problem
```
- I think we should leave out `nodejs` from the query name and ID. It might grow in scope to cover other kinds of streams.
- "Without error handling" sounds a bit off, since there is _some_ error handling, it's just that some of the errors slip through.

Also, if you agree with the query name and ID change, could you rename the `.ql` files to match afterwards?